authui/src/form.ts

```diff
@@ -29,6 +30,7 @@ import { handleAxiosError } from "./messageBar";
 // See https://github.com/authgear/authgear-server/issues/2333
 export class XHRSubmitFormController extends Controller {
```

Comment:
It seems we can revert all changes in this file, as we duplicated one new controller for v2.